python/tank/bootstrap/cached_configuration.py

```diff
@@ -276,6 +281,24 @@ def update_configuration(self):
                     self.path.as_shotgun_dict()
                 )
 
+    def _delete_old_backups(self, latest_backup_folder_path):
+        """
+        Deletes all of the folders in the backups_folder except
+        for the last backup created.
+        """
+    
+        if os.path.exists(latest_backup_folder_path):
+            (backups_folder_path, latest_backup_folder_name) = os.path.split(latest_backup_folder_path)
+            names = os.listdir(backups_folder_path)
+            for name in names:
+                item_path = os.path.join(backups_folder_path, name) 
+                try: 
+                    if os.path.isdir(item_path) and not name == latest_backup_folder_name: 
```

Comment:
do we need to keep the `latest_backup_folder_name`? At this point, we have 

- moved existing config/core into backup 
- installed a new core
- everything worked, the core was successfully installed
- now we perform this cleanup

Do we need to keep the backup at all? Or Is there a use case you can see where we may need it in the future?